tests/runtest.py

```diff
@@ -1,12 +1,36 @@
 #!/usr/bin/env python
-################################################################################
-################################################################################
 #
-# Module: runtest.py
+## Licensed to the .NET Foundation under one or more agreements.
+## The .NET Foundation licenses this file to you under the MIT license.
+## See the LICENSE file in the project root for more information.
+#
+##
+# Title               :runtest.py
 #
 # Notes:
 #  
-# Universal script to setup and run the xunit msbuild test runner.
+# Universal script to setup and run the xunit console runner. The script relies 
+# on runtest.proj and the bash and batch wrappers. All test excludes will also 
+# come from issues.targets. If there is a jit stress or gc stress exclude, 
+# please add GCStressIncompatible or JitOptimizationSensitive to the test's 
+# ilproj or csproj.
+#
```

Comment:
The GCStressIncompatible and JitOptimizationSensitive excludes should probably be called out in docs too since this isn't very discoverable.

Reply:
Addressed